galasa-managers-parent/galasa-managers-logging-parent/dev.galasa.elasticlog.manager/src/main/java/dev/galasa/elasticlog/internal/package-info.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * ElasticLog Manager
+ * 
+ * @galasa.manager ElasticLog
+ * 
+ * @galasa.release.state ALPHA - This Manager has been written to provide a way of exporting test data to ElasticSearch.  It has the bare 
+ * minimum code necessary to do that.  It has not been extensively tested.  The TPI of the LoggingManager interface is subject to change. 
```

Comment:
can remove the TPI and the test can use it.   as tests wont use it.
Maybe "It is available for Galasa administrators to utilise to experiment with Elasic/Kabana dashboards"